homu/main.py

```diff
@@ -394,7 +394,8 @@ def change_treeclosed(self, value, src):
 
     def blocked_by_closed_tree(self):
         treeclosed = self.repos[self.repo_label].treeclosed
-        return treeclosed if self.priority < treeclosed else None
+        return treeclosed if self.priority < treeclosed else None,
+        self.repos[self.repo_label].treeclosed_src
```

Comment:
I'm pretty sure this won't work, the function will just return `(treeclosed if self.priority < treeclosed else None,)`. You need to wrap this in parenthesis.

Reply:
Right. I originally put it on one line then split the line because it was too long.